documentation/staging/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -128,6 +128,36 @@ imagePullSecrets:
 - name: "my-image-pull-secret"
 ```
 
+##### `annotations`
+Specifies a set of key-value annotations that will be added to each pod running the operator. If no customer defined annotations are required, then omit this property.
+
+Example:
+```yaml
+annotations:
+  stage: production
+```
+
+You may also specify annotations [using the "--set" parameter to the Helm install command](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing), as follows:
```

Comment:
"--set" -> Is it possible for this parameter to be in code font, `--set`? (I forget if it does or does not render correctly when inside a x-ref.) If it works, please make the same edit in the `labels` section.

Reply:
Done